### ISSUE TITLE
[Feature] Add an option to collect anonymously data from Alby users of sites with v4v/lightning addresses amd show it on Discover page.

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#2064]
### Feature description

Hi, would be amazing if we have an option to collect anonymously data from Alby users of sites with v4v/lightning addresses. So we could show on Discover page, profiles of twitte, Youtube channels and sites. 
On Discover page could have filter tags like: Twitter profiles, Youtube channels, Sites, Recommended services.

### Describe the solution

Option to Alby extension collect anonymously data of pages Alby users browse and collect and show for all other users the sites with v4v/lightning and show on Discover page as a directory.

### Additional context

Brave has something like this: https://bravebat.info/creators/twitter but I think they use data of verified users to be able to collect the data.

### Are you working on this?

No

**Comment by bitcoinuser** [comment]
And not only v4v/lightning addresses but sites where users used to make lightning payments too.

**Comment by lujakob** [comment]
Probably might be appreciated to explicitly ask if a user wants to "share this v4v website" with other Alby users.

**Comment by bitcoinuser** [comment]
> Probably might be appreciated to explicitly ask if a user wants to "share this v4v website" with other Alby users.

Yes, an option on settings. But I don't see why not if the data will be anonymously sent.
